Formula/cdogs-sdl.rb

```diff
@@ -24,7 +24,7 @@ def install
     system "make"
     prefix.install "src/cdogs-sdl.app"
     bin.write_exec_script "#{prefix}/cdogs-sdl.app/Contents/MacOS/cdogs-sdl"
-    pkgshare.install ["data", "dogfights", "graphics", "missions", "music", "sounds"]
+    pkgshare.install %w[data dogfights graphics missions music sounds]
```

Comment:
Spoke to @ilovezfs in Slack and he seemed to not mind this too much given it's only for string arrays that are >=4 long.